content/webapp/services/prismic/fetch/guides.ts

```diff
@@ -1,10 +1,38 @@
-import { fetcher } from '.';
-import { PagePrismicDocument } from '../types/pages';
+import { Query } from '@prismicio/types';
+import { PrismicQueryOpts } from '@weco/common/services/prismic/types';
+import { fetcher, GetServerSidePropsPrismicClient } from '.';
+import {
+  GuidePrismicDocument,
+  GuideFormatPrismicDocument,
+} from '../types/guides';
+import * as prismic from 'prismic-client-beta';
+import { fetchLinks as pagesFetchLinks } from './pages';
 
 const fetchLinks = [];
 
-const guidesFetcher = fetcher<PagePrismicDocument>('guides', fetchLinks);
+const guidesFetcher = fetcher<GuidePrismicDocument>('guides', fetchLinks);
+const guideFormatsFetcher = fetcher<GuideFormatPrismicDocument>(
+  'guide-formats',
+  []
```

Comment:
could use the variable `fetchLinks` here too

Reply:
I think the fetchLinks for guides and guide formats are different; I can imagine they might diverge at some point, so I'm going to leave this as-is.